dabest/_bootstrap_tools.py

```diff
@@ -9,70 +9,92 @@
 
 class bootstrap:
     '''Computes the summary statistic and a bootstrapped confidence interval.
+
     Keywords:
         x1, x2: array-like
             The data in a one-dimensional array form. Only x1 is required.
             If x2 is given, the bootstrapped summary difference between
             the two groups (x2-x1) is computed.
             NaNs are automatically discarded.
+
         paired: boolean, default False
             Whether or not x1 and x2 are paired samples.
+
         statfunction: callable, default np.mean
             The summary statistic called on data.
+
         smoothboot: boolean, default False
             Taken from seaborn.algorithms.bootstrap.
             If True, performs a smoothed bootstrap (draws samples from a kernel
             destiny estimate).
+
         alpha: float, default 0.05
             Denotes the likelihood that the confidence interval produced
             does not include the true summary statistic. When alpha = 0.05,
             a 95% confidence interval is produced.
+
         reps: int, default 5000
             Number of bootstrap iterations to perform.
+
     Returns:
         An `bootstrap` object reporting the summary statistics, percentile CIs,
         bias-corrected and accelerated (BCa) CIs, and the settings used.
+
         summary: float
             The summary statistic.
+
         is_difference: boolean
             Whether or not the summary is the difference between two groups.
             If False, only x1 was supplied.
+
         is_paired: boolean
```

Comment:
is_paired should no longer be boolean, but we have several instances in the code where is_paired is still boolean/stated as False by default

Reply:
This is already archived file.